DigitalOffice/Tests/FileServiceTests/Repositories/AddNewFileRepositoryTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿using FileService.Database;
+using FileService.Database.Entities;
+using FileService.Repositories;
+using FileService.Repositories.Interfaces;
+using Microsoft.EntityFrameworkCore;
+using NUnit.Framework;
+using System;
+
+namespace FileServiceUnitTests.Repositories
+{
+    public class AddNewFileRepositoryTests
+    {
+        private DbFile newFile;
+        private IFileRepository repository;
+        private FileServiceDbContext dbContext;
+        private DbContextOptions<FileServiceDbContext> dbOptionsProjectService;
+
+        [SetUp]
+        public void Initialization()
+        {
+            dbOptionsProjectService = new DbContextOptionsBuilder<FileServiceDbContext>()
+                .UseInMemoryDatabase(databaseName: "FileServiceTestDatabase")
+                .Options;
+
+            dbContext = new FileServiceDbContext(dbOptionsProjectService);
+            repository = new FileRepository(dbContext);
+
+            newFile = new DbFile
+            {
+                Id = Guid.NewGuid(),
+                Content = Convert.FromBase64String("RGlnaXRhbCBPZmA5Y2U="),
+                Extension = ".txt",
+                IsActive = true,
+                Name = "DigitalOfficeTestFile"
+            };
+        }
+
+        [Test]
+        public void SuccessfulAddNewProjectInDbTest()
+        {
+            Assert.AreEqual(newFile.Id, repository.AddNewFile(newFile));
+        }
```

Comment:
One test?

Reply:
Well, what do you suggest to add here?
Im thinking only about Fail tests

Reply:
But even then all exceptions, checks are passed before dbContext.Add method so i did not include Fail tests.

Could you please name one test that checks something that throws an exception AND that exception would not be thrown by a validator or a mapper that are called before?

Reply:
Hmm. I've come up with some tests concerning Database exceptions. Will be added some time later.

Reply:
Added only one test: Failed Add when that file already exists (duplicate Guid); 
other tests like invalid model or model data I think are unnecessary because these errors will be caught by validators or mappers.

I could add Fail Tests in case someone thinks that we can use repository without addressing the data to Validators and Mappers.